src/gluonts/nursery/autogluon_for_single_ts.py

```diff
@@ -0,0 +1,131 @@
+from autogluon import TabularPrediction as task
+from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.dataset.util import to_pandas
+from pandas.tseries.holiday import USFederalHolidayCalendar as calendar
+import pandas as pd
+import matplotlib.pyplot as plt
+from gluonts.evaluation import Evaluator
+from gluonts.evaluation.backtest import make_evaluation_predictions
+from gluonts.model.forecast import SampleForecast
+from gluonts.dataset.common import ListDataset
+import json
+import numpy as np
+import matplotlib.pyplot as plt
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+class Gluonts3Auto:
```

Comment:
Maybe minor: why this name choice?

Reply:
Sorry. I used to name it as Gluonts2Auto. But after several modifications on the code, I have written Gluonts3Auto, Gluonts4Auto...